### PR TITLE
chore: update to ubuntu:bionic

### PR DESCRIPTION
**Pull request by deepak1556**
* Adds build environment based on Electron's build configuration
* Adds gcc-4.8 toolchain which will maintain compatibility for `GLIBCXX_3.4.19` in remote native modules
* Converts `x64` tag to `trusty-x64` and `bionic-x64`

### CODE REVIEW

File: build.yml [hunk 1]
```diff
@@ -18,4 +18,3 @@ steps:
     repository: 'vscode-linux-build-agent'
     tags: ${{ parameters.arch }}
     command: push
-  condition: and(succeeded(), eq(variables['Build.SourceBranch'], 'refs/heads/master'))
```
Comment (joaomoreno) [inline build.yml:21]: We should not merge this! We only want to push to the registry if the build happens from `master`.

**Review by joaomoreno — APPROVED** [review]
All good, except the `master` condition.

File: build.yml (hunk 1 above)
Comment (deepak1556) [reply build.yml]: Yup was gonna revert before merging, need builds from this branch for testing :)

**Comment by deepak1556** [comment]
Thanks for the review, merging!

**Pull request merged**